valve-monitor/web-app/src/constants/ui.ts

```diff
@@ -12,12 +10,15 @@ const ERROR_MESSAGE = {
     "An internal error occurred. If this problem persists, contact <a href='https://discuss.blues.io' target='_blank' rel='noreferrer'>Blues Support</a>.",
   UNAUTHORIZED:
     "Authentication failed. Please ensure you have a valid HUB_AUTH_TOKEN environment variable.",
-  DEVICE_NAME_CHANGE_FAILED:
-    "An error occurred changing the name.",
+  DEVICE_NAME_CHANGE_FAILED: "An error occurred changing the name.",
   DATABASE_NOT_RUNNING:
     "Can't reach the database server. Please make sure your database is properly connected.",
   NO_PROJECT_ID:
     "A project ID is required, but none was found. Ensure your environment variables are configured correctly for HUB_PROJECTUID.",
```

Comment:
Can you update this error message for NO_PROJECT_ID to say `HUB_PROJECT_UID`?

Reply:
Ah, yep, done.